api/tests/src/db/models/factories.py

```diff
@@ -2454,6 +2454,11 @@ class Params:
         )
 
 
+class StagingTinstructionsFactory(AbstractStagingFactory):
+    class Meta:
+        model = staging.instructions.Tinstructions
+
```

Comment:
Could we add a few fields to this factory so it could be used without having to pass in everything you want?